tests/test_env.py

```diff
@@ -3,36 +3,13 @@
 import pytest
 
 from .. import plugin_parser
-
-class PathDirs:
-    """ Global path directories for parsing templates """
-    def __init__(self,
-                 base_dir=os.getcwd()+"/",
-                 collectors_dir="collectors",
-                 core_dir="core",
-                 plugins_dir="plugins/",
-                 plugin_repos="plugin_repos",
-                 template_dir="templates/",
-                 vis_dir="visualization"):
-        self.base_dir = base_dir
-        self.collectors_dir = base_dir + collectors_dir
-        if not os.path.exists(self.collectors_dir):
-            os.makedirs(self.collectors_dir)
-        self.core_dir = base_dir + core_dir
-        self.plugins_dir = base_dir + plugins_dir
-        self.plugin_repos = base_dir + plugin_repos
-        if not os.path.exists(self.plugin_repos):
-            os.makedirs(self.plugin_repos)
-        self.template_dir = base_dir + template_dir
-        self.vis_dir = base_dir + vis_dir
-        if not os.path.exists(self.vis_dir):
-            os.makedirs(self.vis_dir)
+import test_menu_launcher
```

Comment:
looks like you're only using it for the `PathDirs`, which should probably live in `test_env` that `test_menu_launcher` should import from rather than having it's own